CHANGELOG.md

```diff
@@ -18,6 +18,15 @@ This changelog documents all notable user-facing changes of VAST.
   [#1408](https://github.com/tenzir/vast/pull/1408)
   [@satta](https://github.com/satta)
 
+- 🎁 The schema language now supports algebraic operations on record types:
+     - `record + record`: Combine the fields of 2 records into a new record
+     - `record <+ record`: Like `+`, but prefer the left operand in case of a
+       field name clash
+     - `record +> record`: Like `+`, but prefer the right operand in case of a
+       field name clash
+     - `record - field_name`: Remove the field named `field_name` from a record
+  [#1407](https://github.com/tenzir/vast/pull/1407)
+
```

Comment:
This renders a bit weird, because the PR reference is part of the last nested bullet. Can you rewrite this not to use nested bullets at all so it renders nicely?

Reply:
Oh an please move this to the top.